packages/react-scripts/fixtures/kitchensink/src/App.js

```diff
@@ -1,23 +1,22 @@
-import React from 'react';
+import React, { Component, Children, cloneElement } from 'react';
 
 class BuiltEmitter extends React.Component {
-  constructor(props) {
-    super(props)
-
-    this.callWhenDone = done => done();
+  componentDidMount() {
+    const only = Children.only(this.props.children);
+    if (!Component.isPrototypeOf(only)) {
```

Comment:
To be honest I'd prefer if we just convert it all to class components and always call `notifyRendered` explicitly.